cdap-cli-tests/src/test/java/io/cdap/cdap/cli/CLIMainTest.java

```diff
@@ -713,6 +713,9 @@ public void testMetadata() throws Exception {
     testCommandOutputContains(cli, String.format("get metadata-properties %s", FAKE_DS_ID), "dsKey,dsValue");
 
     // test search
+    testCommandOutputContains(cli, "search metadata \"fake* fake*\" filtered by target-type dataset",
```

Comment:
is there a reason you are using the same search term twice? Seems that it would be better to use two different terms  to be able to validate that both were used by the search

Reply:
The reason I use the same search term twice here is simply to stop the old IllegalArgumentException error that resulted from a space being present in the query. However, it would indeed be more comprehensive to also test the desired functionality. Addressed in the latest commit.